.. _Top:

❗Postdoc❗
==========

Seeking postdoctoral positions starting in Fall 2025
----------------------------------------------------

Introduction
------------

I'm a microbiome data scientist looking for a postdoctoral research position starting fall 2025 to further develop my expertise and contribute to cutting-edge research in the microbiome field. I currently reside in Portland, Oregon but also open to relocating nationally or internationally, or remote-work. 

Research Interests
------------------

I am interested in contributing to research in the following areas:

- **Environment and gut microbiome interaction with host health**: How environmental stressors (e.g. diet, pollutants, pathogens, climate change, etc.) impact gut microbiome stability and resilience to influence host health and homeostasis.
- **Temporal dynamics of host-microbiome systems**: Elucidating temporal patterns in microbiome response to disturbances. 
- **Multi-omic microbiome analysis**: Integrating multivariate multi-omic data to provide a comprehensive view of the gut microbiome and its functional potential.
- **Transdisciplinary Microbiome Research**: Collaborating across disciplines to elucidate the microbiome's central role interfacing between host health, public health and social equity. 

Research Highlights
-------------------

Microbiome and Host Health
""""""""""""""""""""""""""

What: 

- My first project was a multi-factorial microbiome zebrafish study investigating the effects of different commonly used laboratory diets on pathogen exposure sensitivity. 

Why:

- Inconsistency in diet may contribute to inconsistency in microbiome zebrafish study results and challenge our ability to integrate and replicate results across studies.
- Additionally, given that the gut microbiome interfaces with diet and intestinal pathogens, diet may be a key factor in determining host susceptibility to infection.

How: 

- To address this, I fed cohorts of zebrafish one of three commonly used laboratory diets and then exposed them to a pathogen. 
- I then measured the microbiome's response to the diets and pathogen in adult zebrafish at 4 and 7 months of age.

Findings:

- Diet influenced the assembly and successional development of the microbiome, as well as fish's susceptibility to infection.

Outcome: 

- An important implication of this work is that environmental context can confound microbiome-host health associations.
- This work resulted in a peer-reviewed manuscript `(Sieler et al. 2023) <https://rdcu.be/djX1r>`_.

Gut Microbiome Stability and Host Health
""""""""""""""""""""""""""""""""""""""""

What: 

- My second and third PhD dissertation chapters build on my prior work investigating environmental impacts on zebrafish microbiome stability. 

Why: 

- Current methods of microbiome analysis are limited in their ability to detect generalizable temporal patterns in microbiome response to disturbances that are conserved across different host-microbiome systems. 
- So, new methods of multi-omic analysis and temporal stability are needed. 

How: 

- To address this, I'm currently conducting a multi-omic analysis, integrating host transcriptomic and gut microbiome 16S rRNA gene sequence data, to identify potential host and microbiome responses to various combinations of sequential environmental stressors. 
- Additionally, I'm developing novel metrics for evaluating temporal microbiome responses to disturbances that consider changes in microbiome diversity and composition across time. 

Outcome: 

- A key feature of these metrics is that they are system-agnostic, allowing researchers to integrate results across studies and host systems to identify generalizable patterns of microbiome stability/instability to predict health/disease.

Small Molecule Metabolomics Analysis
""""""""""""""""""""""""""""""""""""

What:

- During my Biostatistics and Data Science PhD internship at the Pacific Northwest National Laboratory, I evaluated 12 common batch effect correction algorithms (BECAs) used for small molecule metabolomics analysis. 

Why: 

- Metabolomics comprises the study of various small molecules, including proteins, lipids, amino acids, etc., but often studies treat all of these metabolites similarly during analysis.

How:

- To address this, we conducted a study using a dataset that contained ground truth or no ground truth to compare the performance of 12 common batch effect correction algorithms (BECAs) used for small molecule metabolomics analysis.

Findings:

- I found that particular BECAs were better at resolving unwanted versus wanted variation when ground truth is known or unknown in lipidomic analysis, and these BECAs performance differed to optimal BECAs for other types of metabolites. 

Outcome:

- An important implication of this work is that researchers can use these results to guide optimal BECA selection to improve the accuracy of their analysis.
- This work is currently in-preparation for publication.


Publications
------------

A list of my peer-reviewed publications can be found `here <https://michaelsieler.com/en/latest/Publications/publications.html>`_.


What I offer
------------

- **Adaptable Statistical Programmer** - Proficient in R and other computational tools for analyzing and modeling complex biological datasets.
- **Data-Driven Problem Solver** - Experienced in synthesizing and processing multifactorial, longitudinal datasets to uncover meaningful insights.
- **Effective Science Communicator** - Skilled in translating complex concepts into intuitive data visualizations and compelling graphic designs.
- **Lifelong Learner** - Driven by curiosity and a passion for expanding my expertise in microbiome research and computational biology.
- **Collaborative Thinker** - A strong interdisciplinary team player who thrives in cross-functional environments, bridging microbiology, ecology, and data science.


Future Directions
-----------------

In a postdoctoral research position, I aim to:

- Expand my research on clarifying characteristics of microbiome stability and elucidating the underlying mechanisms that support host health and homeostasis.
- Explore and integrate new multi-omic methodologies for microbiome analysis.
- Collaborate with researchers and institutions across a diverse array of fields to drive innovation and discovery in microbiome science.

Contact Information
-------------------

If you are interested in discussing potential postdoctoral opportunities, please feel free to contact me at:

- `Email <mailto:sielerjm@oregonstate.edu>`_
- `LinkedIn <https://www.linkedin.com/in/mjsielerjr/>`_

I look forward to connecting and exploring potential opportunities in your lab!

Resume & CV
-----------

:download:`Resume (one page) <../ResumeCV/MichaelSieler_Resume.pdf>`

:download:`CV <../ResumeCV/MichaelSieler_CV.pdf>`

------

Return to `top`_.

------