Experience
==========

Summary
-------

* Microbiome data scientist with 5 years of experience developing molecular, computational and statistical approaches to elucidate the underlying mechanisms of the gut microbiome and how they influence host health
* Robust data analytic skills in advanced applied statistics to drive research experiments (R, Python)
* Demonstrated abilities to collaborate and take leadership in multi-laboratory experiments
* Experienced in written and oral communication across various audiences from academics and students to members of the public


Education
---------

+------------------------------------------------------------------------------------------------+--------------------+
| Oregon State University                                                                        | *Expected 2025*    |
+------------------------------------------------------------------------------------------------+--------------------+
| **Ph.D. Microbiology, Minor in Biological Data Sciences**                                                           |
+------------------------------------------------------------------------------------------------+--------------------+
| Oregon State University                                                                        | 2020               |
+------------------------------------------------------------------------------------------------+--------------------+
| **B.Sc. Bioresource Research, Options in Bioinformatics and Genomics**                                              |
+---------------------------------------------------------------------------------------------------------------------+


Research Experience
-------------------

+--------------------------------------------------------------------------------------------------------------------------+
| **Developed high-throughput molecular biological and computational pipelines to interrogate gut microbiome**             |
+--------------------------------------------------------------------------------------------------------------------------+
| * Designed and implemented novel gnotobiotic procedures to process 1,000+ zebrafish embryos to analyze their microbiomes |
| * Tools used: DADA2 (16S processing), R (data cleaning, statistical analysis, visualization)                             |
| * Publications: Stagaman (*in development*)                                                                              |
+--------------------------------------------------------------------------------------------------------------------------+
| **Measure resilience of gut microbiome to chronic exposure of antibiotics**                                              |
+--------------------------------------------------------------------------------------------------------------------------+
| * Exposed 140 adult zebrafish to varying combinations of antibiotics and controls                                        |
| * Tools used: DADA2 (16S processing), R (data cleaning, statistical analysis, visualization)                             |
| * Status: *Currently conducting experiment*                                                                              |
+--------------------------------------------------------------------------------------------------------------------------+
| **Assess gut microbiome resiliency to anthropological impacts such as temperature and pathogenic exposure**              |
+--------------------------------------------------------------------------------------------------------------------------+
| * Exposed adult zebrafish to varying combinations of antibiotics and controls                                            |
| * Tools used: DADA2 (16S processing), R (data cleaning, statistical analysis, visualization)                             |
| * Status: *Waiting on data generation*                                                                                   |
+--------------------------------------------------------------------------------------------------------------------------+
| **Investigate the joint interaction effects of pathogen exposure and diet on gut microbiome succession**                 |
+--------------------------------------------------------------------------------------------------------------------------+
| * Fed 180 zebrafish one of three commonly used laboratory diets and exposed half to a common pathogen                    |
| * Tools used: DADA2 (16S processing), R (data cleaning, statistical analysis, visualization)                             |
| * Publications: Sieler (*in development*)                                                                                |
+--------------------------------------------------------------------------------------------------------------------------+
| **Measure the effect of nanoplastics on the mouse gut microbial community**                                              |
+--------------------------------------------------------------------------------------------------------------------------+
| * Statistically analyzed the effects of nanoplastic exposure on mouse gut microbial communities                          |
| * Tools used: DADA2 (16S processing), R (data cleaning, statistical analysis, visualization)                             |
| * Publications: Szule, *Front. Water* (*in review*)                                                                      |
+--------------------------------------------------------------------------------------------------------------------------+
| **Meta-analysis of zebrafish gut microbiomes phylogeny**                                                                 |
+--------------------------------------------------------------------------------------------------------------------------+
| * Designed and implemented procedures to process 1,000+ zebrafish embryos to analyze their microbiomes                   |
| * Tools used: DADA2 (16S processing), R (data cleaning, statistical analysis, visualization)                             |
| * Publications: `Sharpton 2021, Toxics. <https://bit.ly/3BaF7LX>`_                                                       |
+--------------------------------------------------------------------------------------------------------------------------+
| **Built and maintain Microbial Bioinformatics Hub to collaboratively share microbiome bioinformatic resources**          |
+--------------------------------------------------------------------------------------------------------------------------+
| * Designed and implemented procedures to process 1,000+ zebrafish embryos to analyze their microbiomes                   |
| * Tools used: DADA2 (16S processing), R (data cleaning, statistical analysis, visualization)                             |
| * `Microbial Bioinformatics Hub <https://microbial-bioinformatics-hub.readthedocs.io/en/latest/index.html>`_             |
+--------------------------------------------------------------------------------------------------------------------------+


Work Experience
---------------

+------------------------------------------------------------------------------------------------+--------------------+
| **MJSieler Consulting - Owner**                                                                | May 2022 - Present |
+------------------------------------------------------------------------------------------------+--------------------+
| * Designed, developed, and deployed educational video game software for clients                                     |
| * Educational software used to fulfill grant requirements for communicating scientific research                     |
| * Tools used: C#, Unity                                                                                             |
| * Projects: `Virtual Fish (GitHub) <https://github.com/OSU-Edu-Games/Virtual-Fish>`_                                |
+---------------------------------------------------------------------------------------------------------------------+


Download
--------

:download:`Resume <../ResumeCV/MichaelSieler_Resume[JUL2022].pdf>`
